feat(card): add fillWidth (#4304)
twilio-labs/paste: packages/paste-website/src/pages/components/card/index.mdx

  code={FillHeightExample}
/>

### Fill width

> Card width should fill the available space of the parent container. However, when wrapped in a flex container, the Card will shrink to fit the available space. To prevent this, you can use the `fillWidth` prop.

Reviewer: suggestion/question: Should "flex container" be "flexbox" instead? I feel like the way it's worded right now could get confusing because we have Flex the product and Flex the component that's an implementation of flexbox.

Author: Very good point! I'll make the change

> Card width should fill the available space of the parent container. However, when wrapped in a flex container, the Card will shrink to fit the available space. To prevent this, you can use the `fillWidth` prop.

Reviewer: ```suggestion
By default, Card width fills the available space of the parent container. However, when wrapped in a flexbox, Card shrinks to fit the available space. To prevent this, use the `fillWidth` prop.
```
